tools/bumper/bumper.go

```diff
@@ -115,6 +118,12 @@ func main() {
 			logger.Printf("Bump not needed in component %s", componentName)
 		}
 	}
+
+	logger.Printf("Finished bumping components. resetting repo")
+	err = cnaoRepo.resetRepoInAllowedList(getAllowedList())
+	if err != nil {
+		exitWithError(errors.Wrap(err, "Failed to reset bumping repo"))
+	}
```

Comment:
If we implement this with defer we don't need this call.

Reply:
Done